source/installation/index.rst

```diff
@@ -89,6 +89,13 @@ or the ``mantidworkbench`` package containing the graphical-user-interface
 
    mamba create -n mantid_env -c mantid mantidworkbench
 
+At present we only provide x86 release for OSX. This can still be used by those with Apple Silicon devices (M1, M2). You will need to have Rosetta installed and preceed the setup commands with 'CONDA_SUBDIR=osx-64'.
```

Comment:
```suggestion
For macOS we currently only provide x86 packages. These can still be used by those with Apple silicon devices (M1, M2). You will need to have Rosetta installed and precede the setup commands with 'CONDA_SUBDIR=osx-64'.
```

- use macOS rather than OSX for consistency with the rest of the page)
- reword first sentence to avoid possible ambiguity
- typo fix